components/admin_console/admin_definition.jsx

```diff
@@ -3580,6 +3580,8 @@ export default {
             ],
             schema: {
                 id: 'Audits',
+                name: t('admin.compliance.complianceMonitoring'),
```

Comment:
Unrelated, but super confusing that we use both `name` and `title`.

Reply:
I agree...